modules/nodes-descheduler-installing.adoc

```diff
@@ -12,7 +12,7 @@ The descheduler is not available by default. To enable the descheduler, you must
 * Cluster administrator privileges.
 * Access to the {product-title} web console.
 ifdef::openshift-origin[]
-* Ensure that you have downloaded the link:https://cloud.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in _Obtaining the installation program_ in the installation documentation for your platform.
+* Ensure that you have downloaded the link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in _Obtaining the installation program_ in the installation documentation for your platform.
```

Comment:
The links (previous and changed) do not take me to a site called - Red Hat OpenShift Cluster Manager site. Is this something that needs to be changed?